packages/payload/src/auth/operations/login.ts

```diff
@@ -184,6 +184,7 @@ export const loginOperation = async <TSlug extends CollectionSlug>(
     }
 
     user.collection = collectionConfig.slug
+    user._strategy = 'local-jwt'
```

Comment:
Is this right? Should login via email/password still be considered a JWT strategy?